### DISCUSSION TITLE
R5.0.2 Sunbird ED - Mobile Sign Off is delayed

### DISCUSSION THREAD

**Discussion by PoornimaRajesh** [Sunbird-Ed/Community#222] [General Discussion]
Hi All,

Sunbird ED - Mobile 5.0.2 release is delayed. 

cc:  @luckynvdu @sharathkashyap @swayangjit @chitranshu-keshav
@anunagaraj @ShailandraOjha @Nadiya496

**Comment by anunagaraj** [comment]
@PoornimaRajesh @swayangjit Please share an update on the Mobile sign off.

cc: @luckynvdu @sharathkashyap @chitranshu-keshav
@anunagaraj @ShailandraOjha @Nadiya496

**Comment by anunagaraj** [comment]
@PoornimaRajesh @swayangjit Please share an update on the Mobile sign off. As we have not yet started with Mobile regression and we have 2194 test cases to be executed.

cc: @luckynvdu @sharathkashyap @chitranshu-keshav @ShailandraOjha @Nadiya496

**Comment by sharathkashyap** [comment]
@anunagaraj The Jenkins required some update in the JAVA version for Mobile App to be built. Recently we have upgraded the target sdk version to be compliant with Play Store policies. We have updated the JAVA version to 11. The Build will be released today.

cc: @luckynvdu @chitranshu-keshav @ShailandraOjha @Nadiya496

**Comment by anunagaraj** [comment]
@sharathkashyap We have not received the mobile build yet and the mobile sign off. Please share an update by when we will receive the sign off .

cc: @luckynvdu @chitranshu-keshav @ShailandraOjha @Nadiya496 @PoornimaRajesh @swayangjit

**Comment by anunagaraj** [comment]
@sharathkashyap @swayangjit @PoornimaRajesh Please share an update on the mobile sign off

cc: @luckynvdu @chitranshu-keshav @ShailandraOjha @Nadiya496 @PoornimaRajesh
